embrace-android-delivery/src/main/kotlin/io/embrace/android/embracesdk/internal/delivery/scheduling/SchedulingService.kt

```diff
@@ -13,3 +13,9 @@ interface SchedulingService : CrashTeardownHandler {
      */
     fun onPayloadIntake()
 }
+
+class NoopSchedulingService : SchedulingService {
```

Comment:
Using this in the DeliveryModule for now. If this is not useful in the future when the implementation is fully rolled out, we can delete it then.